### PR TITLE
FAI-8400 | Survey exclude columns

### PR DESCRIPTION
**Pull request by matiaslcoulougian**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Review by tovbinm — APPROVED** [review]

File: destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json [hunk 1]
```diff
@@ -149,6 +149,16 @@
             "{\"Technical Practices\": \"SpeedAndAgility\", \"Planning\": \"AlignmentAndGoals\"}"
           ],
           "order": 5
+        },
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json:152]: We also have a generic exclusion for records' fields - https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/resources/spec.json#L61-L71 

how is this different?

Comment (matiaslcoulougian) [reply destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json:152]: This exclude columns is to exclude row's keys (columns) which are neither metadata nor survey question columns. (Questions that contain extra info we don't wont to save) If they are not filtered, they would be saved as question records.

File: destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json [hunk 2]
```diff
@@ -149,6 +149,16 @@
...
+        "exclude_columns": {
+          "type": "array",
+          "title": "Exclude Columns",
+          "description": "Columns to exclude from the source records so they are not processed as questions.",
+          "items": {
+              "type": "string"
```
Comment (ypc-faros) [inline destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json:158]: nit: weird indentation

**Review by ypc-faros — APPROVED** [review]

File: destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json (hunk 1 above)
Comment (tovbinm) [reply destinations/airbyte-faros-destination/resources/source-specific-configs/surveys.json:152]: well, its not clear from the description.

**Pull request merged**